webpack.config.js

```diff
@@ -134,6 +156,24 @@ const config = {
                     expandProps: 'start'
                 }
             } ]
+        }, {
+            test: /\.(jpg|png)$/,
+            use: [ {
+                loader: 'url-loader',
+                options: {
+                    limit: 11000,
```

Comment:
It will be nice this loader will convert logo to base64 in scss url() rule

Reply:
I guess it's not convert because of `url=false` in `sass-loader` and `css-loader`

Reply:
what's the benefit from that? 
inlining images prevents native browser cache and optimizations